Sources/XMTPiOS/Conversations.swift

```diff
@@ -13,6 +17,20 @@ public actor Conversations {
 		self.client = client
 	}
 
+	public func groups() async throws -> [Group] {
+		try await client.v3Client.conversations().sync()
```

Comment:
We decided to not auto sync groups on groups load and instead groups should take pagination data.